components/net/bluetooth_thread.rs

```diff
@@ -911,10 +918,12 @@ impl BluetoothManager {
         let mut descriptors_vec = vec!();
         for descriptor in descriptors {
             if let Ok(uuid) = descriptor.get_uuid() {
-                descriptors_vec.push(BluetoothDescriptorMsg {
-                                         uuid: uuid,
-                                         instance_id: descriptor.get_id(),
-                                     });
+                if !uuid_is_blacklisted(&uuid, Blacklist::All) {
```

Comment:
I think we should move this check out from the for, and check after we have a list.